mobile/package.json

```diff
@@ -24,9 +24,10 @@
     "tsc": "tsc --init --pretty --jsx react"
   },
   "dependencies": {
-    "@burstjs/core": "^0.0.9",
-    "@burstjs/crypto": "^0.0.9",
-    "@burstjs/http": "^0.0.9",
+    "@babel/runtime": "^7.3.1",
```

Comment:
what's the reason?

Reply:
This was part of the troubleshooting process. It's possible we don't need this anymore.

Reply:
Fixing this now.